Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Multistage docker build, requires docker 17.05
 
 # builder stage
-FROM ubuntu:16.04 as builder
+FROM ubuntu:20.04 as builder
 
 RUN set -ex && \
     apt-get update && \
```

Comment:
<img width="797" alt="Screen Shot 2021-09-11 at 6 25 35 PM" src="https://user-images.githubusercontent.com/3574444/132963058-4068fa40-0a04-49ce-82e4-976be182f913.png">

might be worth setting `DEBIAN_FRONTEND=noninteractive` so that it doesn't block and expect user interaction

Reply:
updated

Reply:
Or should I set it for apt update?

Reply:
lgtm, `install` is the one to go 👍